transformers/avayaVoiceChannel/README.MD

```diff
@@ -0,0 +1,9 @@
+# Description
+Avaya Voice Channel Transformer translates the communication between Avaya CPaaS calls,sms messages and Cognigy's text in real time.
```

Comment:
What is this Avaya service and where can a developer get access and credentials for CPaaS etc.? Please add this information to the doc.

Reply:
made the changes per suggestion